src/core/credentials/transport/tls/grpc_tls_certificate_provider.cc

```diff
@@ -521,7 +518,7 @@ grpc_tls_certificate_provider_file_watcher_create(
   return new grpc_core::FileWatcherCertificateProvider(
       private_key_path == nullptr ? "" : private_key_path,
       identity_certificate_path == nullptr ? "" : identity_certificate_path,
-      root_cert_path == nullptr ? "" : root_cert_path, refresh_interval_sec);
+      root_cert_path == nullptr ? "" : root_cert_path, /*spiffe_bundle_map_path=*/"", refresh_interval_sec);
```

Comment:
Instead of hard-coding the spiffe path to the empty string, we should add a parameter to `grpc_tls_certificate_provider_file_watcher_create()`, so that we can expose this functionality via the C++ TlsCreds API.  Note that this will require a gRFC, so feel free to leave it for a subsequent PR, but in that case please add a TODO.